### PR TITLE
feat(Elife): Add oa and cc icon links

### PR DESCRIPTION
**Pull request by nlisgo**

### CODE REVIEW

File: src/themes/elife/styles.css [hunk 1]
```diff
@@ -366,16 +405,3 @@
     list-style-type: none;
   }
 }
-
-/* Visually hide 'content header' affiliations at narrower viewports, maintaining vertical whitespace */
-@media screen and (--medium-viewport-max) {
```
Comment (nlisgo) [inline src/themes/elife/styles.css:371]: @discodavey was this removal intentional?

Comment (discodavey) [reply src/themes/elife/styles.css:371]: Do not remember removing this

Comment (nlisgo) [reply src/themes/elife/styles.css:371]: > Do not remember removing this. Add it back in if it's needed.

Then I'm adding it back in

**Comment by alex-ketch** [comment]
Copying comment from https://github.com/elifesciences/issues/issues/5878#issuecomment-665065796 for @nlisgo's reference. Could you let me know if you are unable to mark the changes as accepted on Argos CI please?

---

@discodavey The Argos CI is a bit too sensitive and can raise flags for antialiasing artifacts, hence warnings about themes which haven't been actually affected. I will look into if we can reduce the threshold.

Overall though, you should treat them as warnings requiring human review. If things look good can mark the changes as accepted which will make the CI check green.
I believe the Argos CI should be pulling permissions from GitHub, so you and your team should have access once signed in. Could you let me know if that's not the case please? (I've already marked the PR for this issue as accepted as there were no differences reported for the eLife theme).

Screenshot 2020-07-28 at 10 05 39

---

For https://github.com/stencila/thema/pull/210: Argos raised a few false positives, but also highlighted the true change to the eLife theme of adding the `oa` and `cc` icons.

Screenshot 2020-07-28 at 10 03 54

**Comment by discodavey** [comment]
I can see the mark as approved but did not want to approve anything that was to do with the other themes if there were any issues.

Thanks

**Comment by nlisgo** [comment]
@alex-ketch I pressed the Mark as approved button and it didn't do anything.

**Comment by discodavey** [comment]
Same for me

**Comment by alex-ketch** [comment]
Pardon the naive question, but are you logged in to Argos CI (top right button)?
I've marked the changes as accepted, but until I'm able to look into this after the launch please feel free to merge with a failed Argos CI status if the changes are intended or false positives.
Thanks

**Comment by nlisgo** [comment]
> are you logged in

I wasn't 🤦

**Comment by nlisgo** [comment]
Holding off merging this in until after: https://github.com/stencila/thema/pull/216

This will avoid making additional queries to our articles API

File: src/themes/elife/staticCustomProperties.css [hunk 2]
```diff
@@ -49,4 +49,6 @@
   --URL-DOWNLOAD-ICON--NARROWER-VIEWPORTS: url('https://elifesciences.org/assets/patterns/img/icons/download.ecfa2d98.svg');
   --URL-DOWNLOAD-ICON--WIDER-VIEWPORTS: url('https://elifesciences.org/assets/patterns/img/icons/download-full.6691999e.svg');
   --URL-EMAIL-ICON: url('https://elifesciences.org/assets/patterns/img/icons/corresponding-author@1x.89247d49.png');
+  --URL-CC-ICON: url('https://elifesciences.org/assets/patterns/img/icons/cc.ec7b6e9c.svg');
```
Comment (nlisgo) [inline src/themes/elife/staticCustomProperties.css:52]: We should not be referencing assets in this way

Comment (nlisgo) [reply src/themes/elife/staticCustomProperties.css:52]: Will leave it in and address with: https://github.com/elifesciences/issues/issues/5911
